src/solvers/dgsem_tree/dg_1d.jl

```diff
@@ -303,7 +303,7 @@ function calc_volume_integral!(du, u,
                        volume_flux_dg, dg, cache, 1 - alpha_element)
 
     # Calculate FV volume integral contribution
-    fv_kernel!(du, u, equations, volume_flux_fv, dg, cache, element, alpha_element)
+    fv_kernel!(du, u, mesh, equations, volume_flux_fv, dg, cache, element, alpha_element)
```

Comment:
Why the need to pass `mesh` into `fv_kernel!` and then subsequently to `calcflux_fv!`? It doesn't seem to be used there, is it? Same applies to 2D.